.github/workflows/verify-pr-commit.yml

```diff
@@ -274,8 +274,16 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        network: [rococo, mainnet]
+        network: [dev, rococo, mainnet]
         include:
+          - network: dev
+            build-profile: release
+            package: frequency-runtime
+            runtime-dir: runtime/frequency
+            built-wasm-file-name-prefix: frequency_runtime
+            release-wasm-file-name-prefix: frequency-dev_runtime
```

Comment:
I went back and forth on adding it. I don't think we have to verify it, but we are doing the other two there, so thought it might be good so that it gets caught before a release.

Thoughts?

Reply:
Ah you mean just the one line. Yes. I'll remove that.

Reply:
Removed.